README.md

```diff
@@ -55,6 +55,12 @@ To use a different formatter than the one specified in your `.sonarwhalrc` file
 sonarwhal https://example.com --formatters excel
 ```
 
+To use a different rule than the one specified in your `.sonarwhalrc` file:
+
+```bash
+sonarwhal https://example.com --rules html-checker
+```
+
```

Comment:
Maybe we should also add an example for using multiple rules?

Reply:
Updated the readme with a multi rule example. Perhaps we should keep the read me with simple ones and create a wiki page with complete set of examples/options and use cases

Reply:
>  Perhaps we should keep the read me with simple ones 

We could consider moving the information into its own dedicated page.